configuration/configuration.schema.json

```
@@ -0,0 +1,161 @@
+{
+  "title": "Arduino CLI configuration schema",
+  "description": "Describe the parameters available for the Arduino CLI configuration file. This schema should be considered unstable at this moment, it is not used by the CLI to validate input configuration",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "properties": {
+    "board_manager": {
+      "description": "",
+      "properties": {
+        "additional_urls": {
+          "description": "the URLs to any additional Boards Manager package index files needed for your boards platforms.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "build_cache": {
+      "description": "configuration options related to the compilation cache",
+      "properties": {
+        "compilations_before_purge": {
+          "description": "interval, in number of compilations, at which the cache is purged, defaults to `10`. When `0` the cache is never purged.",
+          "type": "number",
+          "format": "integer",
```

```suggestion
          "type": "integer"
```

There is no need to use a [custom](https://json-schema.org/draft-06/json-schema-validation.html#rfc.section.8.3) "integer" format when there is already an "integer" type:

https://json-schema.org/understanding-json-schema/reference/numeric.html#integer